README.md

```diff
@@ -128,6 +128,19 @@ GPU memory.
 Select the version of the TensorFlow library to be used, available
 versions are 1 and 2. Default version is 1.
 
+##### --backend-config=tensorflow,default-max-batch-size=\<int\>
+
+The value to use for max_batch_size during [Autocomplete](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration) when no 
```

Comment:
The default value to use for max_batch_size during [auto-completing model configuration](link)
when batching support is detected in model.

Reply:
Fixed.